plugins/scaffolder-backend/src/scaffolder/stages/publish/publishers.ts

```diff
@@ -51,32 +51,16 @@ export class Publishers implements PublisherBuilder {
 
   static async fromConfig(
     config: Config,
-    { logger }: { logger: Logger },
+    _options: { logger: Logger },
   ): Promise<PublisherBuilder> {
     const publishers = new Publishers();
 
     const scm = ScmIntegrations.fromConfig(config);
 
-    const deprecationWarning = (name: string) => {
-      logger.warn(
-        `'Specifying credentials for ${name} in the Scaffolder configuration is deprecated. This will cause errors in a future release. Please migrate to using integrations config and specifying tokens under hostnames'`,
-      );
-    };
-
     for (const integration of scm.azure.list()) {
       const publisher = await AzurePublisher.fromConfig(integration.config);
       if (publisher) {
```

Comment:
Hm i was hoping that we would be able to remove the undefined return value as well

Reply:
I'll let you take a stab at structuring that :grin: